contrib/coredns/policy/dns_test.go

```diff
@@ -300,6 +332,144 @@ func TestSetRedirectQueryAnswer(t *testing.T) {
 			";; QUESTION SECTION:\n;"+
 			"redirect.example.com.\tIN\t A\n",
 	)
+
+	m = testutil.MakeTestDNSMsg("example.com", dns.TypeA, dns.ClassINET)
+	w = testutil.NewTestAddressedNonwriter("192.0.2.1")
+
+	rc, err = p.setRedirectQueryAnswer(context.TODO(), w, m, "A=127.0.0.1;AAAA=23ef:3546::8732;TXT=Ghguyw7g7yiug7;CNAME=google.com")
+	if err != nil {
+		t.Error(err)
+	}
+	testutil.AssertDNSMessage(t, "setRedirectQueryAnswer(192.0.2.53)", rc, m, dns.RcodeSuccess,
```

Comment:
here and below assertion message like "setRedirectQueryAnswer(192.0.2.53)" should be meaningful, don't copy-paste it blindly everywhere

Reply:
Missed that, will update